src/Host.h

```diff
@@ -215,24 +188,9 @@ class Host : public QObject
     // What the user has set as their preference
     bool mIsCurrentLogFileInHtmlFormat;
 
-    // What the current file will use, set from the previous
```

Comment:
And this one (_before_ lines 218 to 224 inclusive) refers to: `mIsCurrentLogFileInHtmlFormat`.